pkg/mediatype/sniffer.go

```diff
@@ -143,6 +143,36 @@ func SniffBitmap(context SnifferContext) *MediaType {
 	return nil
 }
 
+// Sniffs audio files.
+func SniffAudio(context SnifferContext) *MediaType {
+	if context.HasFileExtension("aac") {
+		return &AAC
+	}
+	if context.HasFileExtension("aiff") {
+		return &AIFF
+	}
+	// TODO flac, m4a
+	if context.HasFileExtension("mp3") {
+		return &MP3
+	}
+	if context.HasFileExtension("ogg", "oga") {
+		return &OGG
+	}
+	if context.HasFileExtension("opus") {
+		return &OPUS
+	}
+	if context.HasFileExtension("wav") {
+		return &WAV
+	}
+	if context.HasFileExtension("webm") {
+		return &WEBMAudio
+	}
+
+	// TODO read magic bytes?
```

Comment:
@mickael-menu I've added the mimetype sniffing for all those audio formats that were missing.
For what you're suggesting though, I'm not sure how it would work. The sniffing functions are always returning specific mimetypes, not a generic audio, video etc. type. What we would return if the `audio/` prefix were detected?

Reply:
Ha you're right, I read too fast and thought this was analog to `MediaType.isAudio` in the Kotlin toolkit.